JYH/4Day/로또의 최고 순위와 최저 순위.js

```diff
@@ -0,0 +1,25 @@
+function solution(lottos, win_nums) {
+    let zerocount = 0;
+    let wincount = 0;
+    
+    const getrank = (count) => {
+        if (count === 6) return 1;
+        if (count === 5) return 2;
+        if (count === 4) return 3;
+        if (count === 3) return 4;
+        if (count === 2) return 5;
+        return 6;
+    };
```

Comment:
```suggestion
const getRank = (count) => [6, 6, 5, 4, 3, 2, 1][count];
```
  위처럼 배열의 인덱스를 count값으로 사용하면 조건문을 반복하지 않고 훨씬 간결하게 작성할 수 있습니다. (+ 카멜케이스 적용)

  그러나, 어떤 상황에선 @jungyoonho128 님께서 작성하신 코드처럼 직관적인 코드가 가독성을 높일 수도 있습니다. 무조건 간결하다고 가독성이 좋은 것이 아니니 적절히 판단해가는 게 좋을 것 같습니다 😀

Reply:
제 나름대로 보기 편한 코드를 만들었는데 짜주신 코드도 정말 간결하고 보기 편한것 같습니다! 알려주셔서 감사합니다